radix-common/src/crypto/signature_validator.rs

```diff
@@ -178,3 +179,21 @@ pub fn fast_aggregate_verify_bls12381_v1(
 
     false
 }
+
+/// Performs BLS12-381 G2 aggregated signature verification
+/// one message signed with multiple keys.
+/// Domain specifier tag: BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_
+/// This method does not validate provided input keys when aggregating,
+/// it is left here for backward compatibility.
+/// It is recommended to use `fast_aggregate_verify_bls12381_v1()` method instead.
+pub fn fast_aggregate_verify_bls12381_v1_old(
```

Comment:
For consistency with elsewhere, I suggest we say where it was introduced, e.g. `fast_aggregate_verify_bls12381_v1_bottlenose` or `fast_aggregate_verify_bls12381_v1_anemone`, can't remember when this was introduced?

Reply:
It was `anemone`, will change 👍